packages/bpk-component-map/src/BpkPriceMarkerV2/BpkPriceMarker.module.scss

```diff
@@ -0,0 +1,73 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '../../../bpk-mixins/index.scss';
+
+.bpk-price-marker {
+  display: flex;
+  height: bpk-spacing-lg();
+  padding: 0 bpk-spacing-md();
+  justify-content: center;
+  align-items: center;
+  border-radius: $bpk-border-radius-sm;
+  gap: bpk-spacing-md();
+
+  &--dynamic {
+    cursor: pointer;
+  }
+
+  &-default {
+    background-color: $bpk-text-on-dark-day;
+    color: $bpk-text-on-light-day;
+
+    &--onDark {
+      background-color: $bpk-text-on-dark-day;
+      color: $bpk-text-on-light-night;
```

Comment:
We shouldn't be using night colours in web. Let me check what we can change this too

Reply:
Hey @olliecurtis Do you have any suggestions? thanks

Reply:
We will need to go back to design as onDark isn't a variant in Figma

Reply:
delete the dark mode